package.json

```diff
@@ -185,7 +185,7 @@
     }
   },
   "lint-staged": {
-    "*!(d).[tj]s": "eslint --fix --report-unused-disable-directives"
+    "*.{js,mjs,cjs,ts,mts,cts}": "eslint --fix --report-unused-disable-directives"
```

Comment:
Didn't know mts and cts files also existed. Does this still ignore .d.ts files?

Reply:
I've un-ignored .d.ts files because the linter is now able to process them. the `types` folder is still ignored, as we're migrating it anyway